lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -848,10 +889,18 @@ instance Arbitrary (ApiTransaction t) where
                 pure Nothing
             (ApiT InLedger) ->
                 arbitrary
+        txPendingSince <- case txStatus of
+            (ApiT Pending) ->
+                arbitrary
+            (ApiT Invalidated) ->
+                pure Nothing
+            (ApiT InLedger) ->
+                pure Nothing
```

Comment:
This is just a nitpick, but perhaps we can save some vertical space here by not inserting as many CRs? So:
```hs
            (ApiT Pending) -> arbitrary
            (ApiT Invalidated) -> pure Nothing
            (ApiT InLedger) -> pure Nothing
```
Though if the parts on the RHS became longer, I do agree that your newline style works better.

Reply:
You should do more `Elm` :smile: ... It'll teach you to appreciate new lines!

( https://github.com/truqu/elm-oauth2/blob/master/src/OAuth.elm )